examples/lusid/holdings/3-create-holdings.sql

```diff
@@ -0,0 +1,41 @@
+-- =====================================================
+-- Description:
+-- 1. In this query, we load holdings into LUSID for the
+-- portfolio.
+-- =====================================================
+
+-- Defining date of file
+@@file_date =
+
+select strftime('20220301');
+
+-- Defining scope and code variables
+@@portfolioScope =
+
+select 'LuminesceHoldingsExample';
+
+@@portfolioCode1 =
+
+select 'UkEquity';
+
+-- Loading data from excel file
+@holdings_from_excel =
+
+use Drive.Excel with @@file_date
+--file=/luminesce-examples/equity_holdings_{@@file_date}.xlsx
+--worksheet=lusid_holdings
+--addFileName
+enduse;
+
+-- Run holdings transformation
+@holdings_for_upload =
+
+select @@portfolioCode1 as [PortfolioCode], @@portfolioScope as [PortfolioScope], holding_date as
```

Comment:
Is this a **set** or **adjust** by default? 
Add the WriteAction parameter which explicitly sets the holding?

Reply:
This file has been removed and as the new `3-set-holdings.sql` performs identically just with the explicit SET value.